### DISCUSSION TITLE
Sprites on 320x240 Display

### DISCUSSION THREAD

**Discussion by SkydiveH** [Bodmer/TFT_eSPI#2941] [Q&A - Sprites]

**Comment by SkydiveH** [comment]
Hello sir,
I'm using the library very extensive and it is working perfectly for my Purpose. The only problem I have is if I want to create a sprite greater than 60x260. I tried 80x260 and when I request if he has created the sprite he denies it, he brings back 0 . I found that there is a limit for sprites greater than appr. 200x100.  60x260 works, 80x260 don't.
I'm using the Arduino IDE 2.2.1 with the latest update of the TFT_eSPI. Is there a possibility to create sprite bigger then the mentioned above or so. I don't not know where the effective limit is and how to change that. The biggest sprite I wanted to create was 240x320 and that don't worked, so I have to distribute it in separate sprite. That worked but is a not so satisfactory . 
The processor I'm using is a ESP32 Dev Kit 4 with 16M. Memory space should be not the problem but how to enable sprite greater than the 200x100?
Thank you 
Skydive

**Comment by Bodmer** [comment]
To create larger sprites the board will need to have onboard PSRAM and that must be enabled in the IDE. The library will then use PSRAM for all sprites.

**Reply by SkydiveH** [reply]
Thank you,
That answers my question.
Skydiveh
